Updates to prometheus and node exporter

sabre1041 opened the pull request:
Updates to prometheus and node exporter

@wkulhanek please review

wkulhanek commented:
@sabre1041 doesn't iptables-save mess up the whole firewall with then persistent rules for any service that might be defined? That is usually a big problem... and why I added the rule manually. 

Also OPENTLC environments only have 4Gi volumes. And finally Brian Brasil said in an OpenShift Commons briefing that Prometheus is designed for ephemeral storage. So maybe we don't need a PVC at all.

sabre1041 commented:
@wkulhanek Updated PR

Went through and updated README based on [this](https://access.redhat.com/solutions/2998541) solutions article.

Regarding persistent storage, I have experienced a few issues while testing related to PV's and explained in [this](https://github.com/prometheus/prometheus/issues/2805) issue. Removed use of PV's in prometheus template to use _emptyDir_
